addons/mass_mailing/models/mailing.py

```diff
@@ -131,12 +131,19 @@ def _get_default_mail_server_id(self):
         default=_get_default_mail_server_id,
         help="Use a specific mail server in priority. Otherwise Odoo relies on the first outgoing mail server available (based on their sequencing) as it does for normal mails.")
     contact_list_ids = fields.Many2many('mailing.list', 'mail_mass_mailing_list_rel', string='Mailing Lists')
+    # A/B Testing
+    ab_testing_description = fields.Html('A/B Testing Description', compute="_compute_ab_testing_description")
+    ab_testing_done = fields.Boolean(related='campaign_id.ab_testing_done', store=True)
+    ab_testing_mailings_count = fields.Integer(related="campaign_id.ab_testing_mailings_count")
+    ab_testing_schedule_datetime = fields.Datetime(related="campaign_id.ab_testing_schedule_datetime", readonly=False)
+    ab_testing_winner_selection = fields.Selection(related="campaign_id.ab_testing_winner_selection", default="opened_ratio", readonly=False, copy=True)
     contact_ab_pc = fields.Integer(string='A/B Testing percentage',
```

Comment:
While we are at it, why not renaming `contact_ab_pc` to `ab_testing_pc` ? That way all chaussettes are well ranged.